Stripe/PublicHeaders/STPSource.h

```diff
@@ -140,6 +131,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable, readonly) STPSourceWeChatPayDetails *weChatPayDetails;
 
+#pragma mark - Deprecated
+
+/**
+ A set of key/value pairs associated with the source object.
+ 
+ @deprecated Metadata is no longer returned to clients using publishable keys. Retrieve them on your server using you secret key instead.
+ 
+ @see https://stripe.com/docs/api#metadata
+ */
+@property (nonatomic, nullable, readonly) NSDictionary<NSString*, NSString *> *metadata DEPRECATED_MSG_ATTRIBUTE("Metadata is no longer returned to clients using publishable keys. Retrieve them on your server using you secret key instead.");
```

Comment:
```suggestion
@property (nonatomic, nullable, readonly) NSDictionary<NSString*, NSString *> *metadata DEPRECATED_MSG_ATTRIBUTE("Metadata is no longer returned to clients using publishable keys. Retrieve them on your server using your secret key instead.");
```

This typo is copied in a handful of places